vibrato/src/dictionary.rs

```diff
@@ -133,6 +133,10 @@ impl Dictionary {
 
     /// Creates a dictionary from a reader.
     ///
+    /// Unlike CLI tools, this function requires an uncompressed dictionary, so you need to use a
+    /// zstd decoder such as [zstd crate](https://docs.rs/zstd/latest/zstd/) or [ruzstd
+    /// crate](https://docs.rs/ruzstd/latest/ruzstd/) to decompress the dictionary beforehand.
```

Comment:
```suggestion
    /// This function requires a dictionary in uncompressed format.
```
I would like to minimize the dependency between the `vibrate` crate and the outside as much as possible. The zstd format of distributed models is just one example and should not inherently be related to this API.

How about my suggestion?